app/Kernel.php

```diff
@@ -71,6 +73,21 @@ public function boot()
     }
 
     /**
+     * We need to manually initialize SpoonSession for the Symfony sessions to work properly
+     */
+    private function fixSymfonySessions()
+    {
+        SpoonSession::start();
+
+        /** @var \Symfony\Component\HttpFoundation\Session\Session $session */
+        $session = $this->getContainer()->get('session');
```

Comment:
@tijsverkoyen this should be fixed in https://github.com/forkcms/forkcms/pull/2023/commits/3513b16b6ca4b30bb38a87bbe4194f98ff73b032